src/UserManagement/List.tsx

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
 import User from './User';
+import useApi from '../hooks/useApi';
 
-const List = () => (
-  <>
-    <h1>List</h1>
-    <ul>
-      <li><User name="Anna" /></li>
-      <li><User name="Marco" /></li>
-      <li><User name="Vivi" /></li>
-      <li><User name="Matteo" /></li>
-    </ul>
-  </>
-);
+type Player = {
+  _id: string,
+  name: string,
+  picture: string | null
+}
+
+const List = () => {
+  const [error, players] = useApi<Player[]>('/players');
```

Comment:
Das hier ist die eigentliche Einbindung der Hook. Das ist jetzt nur ein Beispiel, aber so würde es dann auch in anderen Komponenten funktionieren. Die Hook gibt ein Tupel `[error, result]` zurück - wenn der Request fehlschlägt, ist `error` gesetzt, andernfalls ist `result` gesetzt.

Reply:
Sidenote: Sorry wenn es hier merge-Konflikte gibt. Ihr könnt das gerne alles zurücksetzen hier, ich wollte es nur irgendwo einbauen, damit man sieht, wie die Hook generell funktioniert.

Reply:
Und noch ein Kommentar dazu, sorry für Spam: Das hier wird automatisch geupdated, sobald der Request durch ist. Das heißt, anfangs ist `error` und `players` hier `null`. Dann geht der Request durch, damit wird dann `error` oder `result` gesetzt. Sobald das passiert, wird durch React automatisch erneut gerendet (wegen der `useState()` Hook innerhalb der `useApi()` Hook). Obwohl sich das also linear liest, ist es ja ein asynchroner Prozess, der dann auch erneut rendert, wenn sich was ändern sollte.